integration_test/third_party_apps_data/applications/aerospike/sles/install

```diff
@@ -3,19 +3,33 @@ set -e
 # aerospike-init requires `python` in $PATH
 sudo ln -s /usr/bin/python3 /usr/bin/python
 
-curl -L -o aerospike.tar.gz https://github.com/aerospike/aerospike-server/releases/download/5.7.0.23/aerospike-server-community-5.7.0.23.tar.gz
+if [[ "$(uname -m)" == aarch64 ]]; then
+    # 6.2 is the first version with ARM64 support
+    curl -L -o aerospike.tar.gz https://github.com/aerospike/aerospike-server/releases/download/6.2.0.8/aerospike-server-community-6.2.0.8.tar.gz
+else
+    curl -L -o aerospike.tar.gz https://github.com/aerospike/aerospike-server/releases/download/5.7.0.23/aerospike-server-community-5.7.0.23.tar.gz
+fi
 tar -xzf aerospike.tar.gz
 cd aerospike-server
 ./bin/aerospike init
 
 # aerospike-init creates an invalid config
 sed -i 's|default-ttl.*||g' etc/aerospike.conf
+if [[ "$(uname -m)" == aarch64 ]]; then
+    sed -i 's|paxos-single-replica-limit.*||g' etc/aerospike.conf
+fi
 
 sudo ./bin/aerospike start
 ./bin/aerospike status
 
 cd -
 
-wget -O aerospike-tools.tgz "https://download.aerospike.com/artifacts/aerospike-tools/7.1.1/aerospike-tools-7.1.1-el8.tgz"
-tar -xzf aerospike-tools.tgz
-(cd aerospike-tools-7.1.1-el8 && sudo ./asinstall)
+if [[ "$(uname -m)" == aarch64 ]]; then
```

Comment:
could we factor out the version? this would reduce duplicated code and also make it easier for us to update the version going forward.

```
if [[ "$(uname -m)" == aarch64 ]]; then
    TOOLS_VERSION=8.1.0
else
    TOOLS_VERSION=7.1.1
fi
wget -O aerospike-tools.tgz "https://download.aerospike.com/artifacts/aerospike-tools/${TOOLS_VERSION}/aerospike-tools_${TOOLS_VERSION}_el8_aarch64.tgz"
    tar -xzf aerospike-tools.tgz
    (cd aerospike-tools_${TOOLS_VERSION}_el8_aarch64 && sudo ./asinstall)
```

same with lines 6-11.

Reply:
oh I missed that. All right, never mind then. Lines 6-11 could still use it though IMO.